bin/kafka-run-class.sh

```diff
@@ -140,6 +140,11 @@ do
 done
 shopt -u nullglob
 
+if [ -z "$CLASSPATH" ] ; then
+  echo 'Classpath is empty. Please build the project first e.g. by running ''./gradlew jar -Pscala_version=$SCALA_VERSION'''
```

Comment:
The script itself sets the variable if unset by the user. So, it seems safe to print it.